ide-support/revsaveasstandalone.livecodescript

```diff
@@ -660,7 +665,11 @@ command revSaveAsMacStandalone pStack, pRestoreFileName, pFolder, @xStandalonePa
    
    -- Architectures
    local tArchitectures
-   if sStandaloneSettingsA["MacOSX x86-32"] then put "i386," after tArchitectures
+   if sStandaloneSettingsA["MacOSX x86-32"] then 
+      delete variable sStandaloneSettingsA["MacOSX x86-32"]
+      put true into sStandaloneSettingsA["MacOSX x86-64"]
+   end if
```

Comment:
@livecodepanos do we need this? It looks like this check is already performed in revSaveAsStandalone

Reply:
@livecodeian Indeed this was not needed, I have now removed it. Thanks :)